client/my-sites/media-library/list.jsx

```diff
@@ -153,9 +154,8 @@ export const MediaLibraryList = React.createClass( {
 		return 'item-' + item.ID;
 	},
 
-	renderItem: function( item ) {
-		var index = findIndex( this.props.media, { ID: item.ID } ),
-			selectedItems = this.props.mediaLibrarySelectedItems,
+	renderItem: function( item, i, colIndex ) {
+		var selectedItems = this.props.mediaLibrarySelectedItems,
```

Comment:
I know it's legacy code, but It may be a good time to get rid of theses `var`s  and use `const` or `let`.